docs/index.rst

```diff
@@ -0,0 +1,52 @@
+.. djangocms-pageadmin documentation master file, created by
+   sphinx-quickstart on Thu Feb 14 11:45:02 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to djangocms-pageadmin's documentation!
+===============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+Overview
+--------
+
+Django CMS Page Admin provides a new PageContent admin which doesn't
+include tree functionality. It also displays data from Django CMS Versioning
+and Django CMS Version Locking.
```

Comment:
I think it's worth adding something here about the fact that unlike the standard CMS page tree, it uses a standard django admin view which allows the addition of actions, filters and so forth.

Is it worth mentioning that it was created to compliment djangocms-navigation? I think so.